data/saas/saas_connector_registry.toml

```diff
@@ -67,8 +67,8 @@ human_readable = "Mailchimp"
 [mailchimp_transactional]
 config = "data/saas/config/mailchimp_transactional_config.yml"
 dataset = "data/saas/dataset/mailchimp_transactional_dataset.yml"
-icon = "data/saas/icon/default.svg"
-human_readable = "Mailchimp Transactional"
+icon = "data/saas/icon/mailchimp.svg"
+human_readable = "Mailchimp Transactional (Mandrill)"
```

Comment:
![Screen Shot 2023-02-01 at 3 48 16 PM](https://user-images.githubusercontent.com/9755598/216170985-44b58c3d-e1e1-4b6c-8406-a751451c3152.png)

I updated to Mailchimp Transactional (Mandrill) to further distinguish from Mailchimp, but added the Mailchimp logo.  Is the name too long now? What do you think?

Reply:
@pattisdr 
Maybe we can use just "Mandrill" and update the logo the mandrill logo?

Reply:
@saiatethyca Mandrill logo updated, I can still change name to "Mandrill" if you think this human readable name is too long.

<img width="961" alt="Screen Shot 2023-02-01 at 4 57 47 PM" src="https://user-images.githubusercontent.com/9755598/216189974-0021ca15-996d-410a-84e8-68fa3d8b0c70.png">